leaf-ui/rappid-extensions/AnalysisInspector.js

```diff
@@ -23,67 +21,9 @@ var AnalysisInspector = Backbone.View.extend({
 		'</select>',
 		'<label class="sub-label">Num Relative Time Points</label>',
 		'<input id="num-rel-time" class="analysis-input" type="number" min="0" max="20" step="1" value="1"/>',
-		'<label class="sub-label">Absolute Time Points</label>',
-		'<font size="2">(e.g. 5 8 22)</font>',
-		'<input id="abs-time-pts" class="analysis-input" type="text"/>',
 		'<hr>',
-		'<button id="btn-view-assignment" class="analysis-btns inspector-btn sub-label green-btn">View List of Assignments</button>',
 		'<button id="btn-view-intermediate" class="analysis-btns inspector-btn sub-label green-btn">View Intermediate Values</button>',
-
-		// This is the modal box of assignments
-		'<div id="myModal" class="modal">',
```

Comment:
is this the myModal thing I mentioned earlier? looks like you deleted, so what does the var in onFunctions refer to?

Reply:
Yes, this is the same thing. I deleted it in AnalysisInspector.js to move it into the index.html since that is where the html for the toolbar is located. The functions for AnalysisInspector are in the same file, but the functions for the toolbar are in onFunctions.js.